### PR TITLE
Update versioning information for dependencies, baseline & CI

### PR DESCRIPTION
**Pull request by LinqLover**
This PR updates various versioning information for the project. Updated the baseline in order to make the whole project installable by using Metacello only, without installing any dependencies or SAR files separately. Refactored the CI configuration to remove some duplication and use Squeak-5.2, 5.3, and Trunk respectively.

Also increases timeouts for authentication handler test logic because in many cases, the authentication did not happen yet on CI before the tests timed out.

Adds description of installation via Metacello to the `README.md` (1be3f0f). These instructions already have been added to the wiki via https://github.com/hpi-swa-teaching/TelegramClient/wiki/Setup-Guide/3364dad29a019b66be4b266c3b528535016184b3.

Last but not least: Fixes a failing test on master due to assumed change on the server side (8170a51).

Might be squashed when merging.

### CODE REVIEW

**Comment by LinqLover** [comment]
- [ ] TODO after merging: Update required settings here: https://github.com/hpi-swa-teaching/TelegramClient/settings/branches

**Comment by LinqLover** [comment]
@codeZeilen @Mrnikbobjeff Would someone of you like to give this PR a review? :-)

File: .github/workflows/ci-linter.yml
Comment (Mrnikbobjeff) [inline .github/workflows/ci-linter.yml:19]: We had trunk as the default target as well, but as trunk is not stable enough for a ci (we had to merge without trunk approval quite often) we opted for a stable version.

File: .github/workflows/main.yml
Comment (Mrnikbobjeff) [inline .github/workflows/main.yml:3]: Without the branch filter we would have ci runs on branches without pull request which are not master or develop

File: .github/workflows/main.yml
Comment (Mrnikbobjeff) [inline .github/workflows/main.yml:12]: Our tests sometime failed IIRC when several CI's were running at the same time. No idea if we ever fixed it, but this was the reason for only one CI image

File: .github/workflows/release.yml
Comment (Mrnikbobjeff) [inline .github/workflows/release.yml:4]: This mistags releases, trunk release did not refer to the squeak version but to our trunk (develop). now master releases are also classified as trunk releases even though they are not

**Review by Mrnikbobjeff — CHANGES_REQUESTED** [review]
Looks good, some things should be touched again

File: .github/workflows/main.yml
Comment (LinqLover) [reply .github/workflows/main.yml:12]: Yeah, I have experienced this issue, too, and fixed it via 8170a51.

File: .github/workflows/main.yml
Comment (LinqLover) [reply .github/workflows/main.yml:3]: Hm, what would be the problem with that? If another branch is chosen for any kind of development/testing, CI results are always helpful IMHO.

File: .github/workflows/ci-linter.yml
Comment (LinqLover) [reply .github/workflows/ci-linter.yml]: Fair point. Resolved via 3a6cd1f.

File: .github/workflows/release.yml
Comment (LinqLover) [reply .github/workflows/release.yml:4]: Note to myself: CI is green again, but I need to test the release process manually like in aa0d83f268fcdffd0a4f094f1a9f102b38260d59 before this can be merged. I hope I'll find some time for this soon ...

File: .github/workflows/main.yml
Comment (Mrnikbobjeff) [reply .github/workflows/main.yml:3]: Well you would have to look quite hard in comparison to find CI results for the branch. Also, if you would have to pay for that one would reduce the amount of ci runs being started.

**Comment by Mrnikbobjeff** [comment]
Could we perhaps also close #308 while we are meddling with the release process. Simply ammed the time format specifier to include minutes and seconds and we should be good to go for our project size

File: .github/workflows/main.yml
Comment (LinqLover) [reply .github/workflows/main.yml]: Fair point. There are worker limits for the CI although I never have exceeded them so far. Resolved via 907ea55. :-)

File: .github/workflows/release.yml
Comment (LinqLover) [reply .github/workflows/release.yml]: Should be implemented via abd0997? See https://github.com/LinqLover/TelegramClient/releases/tag/Trunk20201101-141029 for example.

**Comment by LinqLover** [comment]
> Could we perhaps also close #308 while we are meddling with the release process. Simply ammed the time format specifier to include minutes and seconds and we should be good to go for our project size

Implemented in 61f56c8 :-)

**Review by Mrnikbobjeff — APPROVED** [review]
Looks good

**Comment by LinqLover** [comment]
Great, thanks for your time! :-)

**Pull request merged**
